src/SDKs/ManagedServices/Management.ManagedServices/Properties/AssemblyInfo.cs

```diff
@@ -1,19 +1,19 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Reflection;
 using System.Resources;
+using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("Microsoft Azure ManagedServices Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure ManagedServices management functions for managing the Microsoft Azure ManagedServices service.")]
-
-[assembly: AssemblyVersion("0.9.0")]
-[assembly: AssemblyFileVersion("0.9.0")]
+[assembly: AssemblyTitle("Microsoft Azure Managed Services Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Maanged Services.")]
 
+[assembly: AssemblyVersion("0.0.9.0")]
```

Comment:
This should be bumped to `1.0.0.0` as well

Reply:
Ok